cli/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "ng-universal-demo",
+  "version": "0.0.0",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/angular/universal-starter.git"
+  },
+  "contributors": [
+    "AngularClass <hello@angularclass.com>",
+    "PatrickJS <patrick@angularclass.com>",
+    "Jeff Whelpley <jeff@gethuman.com>",
+    "Jeff Cross <crossj@google.com>",
+    "Mark Pieszak <mpieszak84@gmail.com>",
+    "Jason Jean <jasonjean1993@gmail.com>",
+    "Fabian Wiles <fabian.wiles@gmail.com>"
+  ],
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "build:prerender": "ng build --prod && ng build --prod --app 1 && node prerender",
```

Comment:
What if we use `build:universal` instead of `build:prerender`?

For now, people know Universal with a different name and there is not yet any official title for it after Universal merged with Angular, we know it as prerender, server-side rendering, platform-server and Universal.
Maybe it helps to reduce the chance of confusing people by using `universal` instead of `prerender`

Reply:
Both use cases come under Universal.
So I think dynamic/static might be better names